src/context/Global.tsx

```diff
@@ -115,6 +117,29 @@ const migrateSwapsFromLocalStorage = async () => {
     return migratedSwapCount;
 };
 
+const migrateToChainSwaps = async (swapsForage: any) => {
+    const swaps = await swapsForage.keys();
+    let migratedSwapCount = 0;
+    for (const swapId of swaps) {
+        const swap = await swapsForage.getItem(swapId);
+        if (!("type" in swap)) {
+            if (swap.reverse) {
+                swap.type = SwapType.Reverse;
+                swap.assetSend = LN;
+                swap.assetReceive = swap.asset;
+            } else {
+                swap.type = SwapType.Submarine;
+                swap.assetSend = swap.asset;
+                swap.assetReceive = LN;
+            }
```

Comment:
Did you ever test this? In progress swaps need to go through and there are bunch of other properties we renamed (eg keys)